pages/api/articles/[articleId].ts

```diff
@@ -0,0 +1,11 @@
+import {Article} from "models";
+import type {NextApiRequest, NextApiResponse} from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const {articleId} = req.query;
```

Comment:
```suggestion
  const { articleId } = req.query;
```
Si estás trabajando con VSCode, `option + shift + O` formatea el archivo y remueve imports que no se usan.

Reply:
Excelente bro, tengo que  actualizar el coding style de WebStorm hehe.